database/db_connection.js

```diff
@@ -0,0 +1,21 @@
+const {Pool} = require('pg');
+const url = require('url');
+require('env2')('./config.env');
+
+if (!process.env.DB_URL) throw new Error('Enviroment variable DB_URL must be set');
```

Comment:
We just need to make sure the database url that heroku uses has the variable name DB_URL (it defaults as DATABASE URL)